recipe/recipe.yaml

```
@@ -1,18 +1,16 @@
 schema_version: 1
 
 context:
-  version: "1.8.0"
-  # sometimes used to match PyPI releases
-  # NOTE: rattler-build does not support empty strings, therefore we use
-  # whitespace that we trim to get an empty string
-  tag: " "
+  full_version: "1.8.0"
+  # X.Y.Z-N used in rare cases for PyPI releases
+  version: ${{ (full_version | split("-"))[0] }}
```

this still won't work; the bare variable (here `full_version`) is the one that needs to be used in the URL. However, I believe it actually needs to be named `version` to work
```suggestion
  version: "1.8.0"
  # X.Y.Z-N used in rare cases for PyPI releases
  version_sanitized: ${{ (version | split("-"))[0] }}
```